lib/Screens/Login.dart

```diff
@@ -1,158 +1,159 @@
 import 'package:flutter/material.dart';
 import 'package:font_awesome_flutter/font_awesome_flutter.dart';
+import 'package:food_ordering/utils/utils.dart';
 
-class SignIn extends StatelessWidget {
+class Login extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
-    return Container(
-      decoration: BoxDecoration(
-        image: DecorationImage(
-          // Background Image
-            image: AssetImage('Assets/pizza.jpg'), 
-            fit: BoxFit.cover),
-      ),
-      child: Scaffold(
-        backgroundColor: Colors.transparent,
-              body: Container(
-          width: MediaQuery.of(context).size.width,
-          decoration: BoxDecoration(
-            borderRadius: BorderRadius.circular(20),
-            color: Colors.transparent,
-          ),
-          child: Padding(
-            padding: const EdgeInsets.all(23.0),
-            child: ListView(
-              shrinkWrap: true,
-              children: <Widget>[
-                SizedBox(height: 200,),
-                Form(
-                  child: Column(
-                    children: <Widget>[
-                      Padding(
-                        padding: const EdgeInsets.fromLTRB(0, 20, 0, 20),
-                        child: TextFormField(
+    return Scaffold(
+      body: Container(
+        decoration: BoxDecoration(
+          image: DecorationImage(
+              image: AssetImage('assets/pizza.jpg'), fit: BoxFit.cover),
+        ),
+        child: Scaffold(
+          backgroundColor: Colors.transparent,
+          body: Container(
+            width: MediaQuery.of(context).size.width,
+            decoration: BoxDecoration(
+              borderRadius: BorderRadius.circular(20),
+              color: Colors.transparent,
+            ),
+            child: Padding(
+              padding: const EdgeInsets.all(23.0),
```

Comment:
keep sucha a constants in some seperate file like with Strings so we can reuse them later in other screens

Reply:
I don't think that we need to declare padding in a different file, cause the amount of code will remain the same

Reply:
But we will use same vertical padding across all screens, so for now it doesn't improve much, but later it would help to change whole app padding in one place.

Reply:
Ok will do it